app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,10 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = Comment.new(user_id: current_user.id, post_id: params[:post_id], text: params[:text])
+    if @comment.save
+      redirect_to user_post_path(:user_id, :post_id), notice: 'Your comment has been successfully created.'
+    else
+      redirect_to user_post_path(:user_id, :post_id), alert: 'Error creating comment.'
+    end
+  end
+end
```

Comment:
- [ ] There should be a form in the views/comments/new.html.erb file that accepts input for creating comments as per requirement, kindly refactor your folder and file structure to meet this requirement.

Reply:
As stated above, as it belongs to current_user method and not included in a project Requirement.

![image](https://github.com/zunairkhan811/blog-app/assets/116083031/6677b939-897e-4ba0-a110-cdedbf64446d)

Reply:
@zunairkhan811, Kindly take note that you still need to create a form for comments as seen in the project requirements:

<img width="646" alt="image" src="https://github.com/zunairkhan811/blog-app/assets/52934128/2aa22aa7-680f-44d0-80d9-ce388d054e74">